src/core/Akka/Configuration/Hocon/HoconObject.cs

```diff
@@ -161,6 +161,7 @@ public string ToString(int indent)
             var sb = new StringBuilder();
             foreach (var kvp in Items)
             {
+                if (kvp.Value.AdoptedFromFallback) continue;
```

Comment:
Got it, your intellisense comment was very helpful. So we use this flag to skip objects that were being merged in from a fallback?